IngresseSDK/Model/Planner.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Planner.swift
+//  IngresseSDK
+//
+//  Created by Rubens Gondek on 12/5/17.
+//  Copyright © 2017 Ingresse. All rights reserved.
+//
+
+public class Planner: JSONConvertible {
```

Comment:
Must add @objcMembers for swift 4.

Reply:
It's added on super class, JSONConvertible is marked with @objcmembers already and all of its subclasses conforms to it as well